klippy/extras/probe.py

```diff
@@ -223,9 +223,11 @@ def get_position_endstop(self):
 # position at each point.
 class ProbePointsHelper:
     def __init__(self, config, finalize_callback, default_points=None):
+        self.config = config
```

Comment:
Thanks.  We really don't want to store a reference to the `config` object, because accessing it after the config phase could result in unexpected behaviour.  Avoid storing the reference by using `raise self.printer.config_error('xxx')` instead.

-Kevin

Reply:
Thanks   that did feel a tad icky.  fixed.